resources/shaders/gltwodshader.frag

```diff
@@ -2,11 +2,24 @@
 
 in vec4 colour;
 in vec2 texuv;
+flat in uint paletteid;
 
 out vec4 FragColour;
 
 uniform sampler2D texture0;
+uniform usamplerBuffer palbuf;
+uniform bool repaint;
 
 void main() {
-    FragColour = texture(texture0, texuv) * colour;
+    vec4 fragcol = texture(texture0, texuv);
+    uint index = uint(fragcol.r * 255);
+    uvec4 newcol = texelFetch(palbuf, int( 256 * paletteid + index));
+
+    if (repaint == true) {
+	if (paletteid > 0)
+	    if (index > 0)
+	        fragcol = vec4(newcol.r / 255.0, newcol.g / 255.0, newcol.b / 255.0, 1.0);
```

Comment:
560ti under linux doesn't like division of uvec4 and saving it into vec4:
```
0(21) : error C1020: invalid operands to "/"
0(21) : error C1020: invalid operands to "/"
0(21) : error C1020: invalid operands to "/"
(0) : error C2003: incompatible options for link
```
changing newcol to vec4 fix this.